src/tests/utests/iotc_utest_bsp_crypto_ecc.c

```diff
@@ -37,35 +37,33 @@ static const iotc_crypto_private_key_data_t DEFAULT_PRIVATE_KEY = {
     .private_key_signature_algorithm =
         IOTC_JWT_PRIVATE_KEY_SIGNATURE_ALGORITHM_ES256,
     .private_key_union_type = IOTC_CRYPTO_KEY_UNION_TYPE_PEM,
-    .private_key_union.key_pem.key =
-        "\
+    .private_key_union.key_pem.key = "\
```

Comment:
This file contains only formatting changes. I'd suggest to keep the scope of PR clear, remove these changes.

Reply:
Is this file even needed? This is replaced by its .cc counterpart, no?

Reply:
Yes, i rewrote it as .cc.
I think this change just came from format change on save.